freestyle-fs2/shared/src/main/scala/fs2.scala

```diff
@@ -1,35 +1,33 @@
 package freestyle
 
 import _root_.fs2._
-import _root_.fs2.util.{Attempt, Catchable, Suspendable, Free}
+import _root_.fs2.util.{Attempt, Catchable, Suspendable, Monad, Free}
 
-import cats._
+import cats.{MonadError, Monoid, ~>}
 import cats.free.{Free => CFree}
-import cats.arrow.FunctionK
-
-import scala.concurrent._
 
 object fs2 {
   type Eff[A] = Free[CFree[Attempt, ?], A]
 
-  implicit val catsFreeC: Catchable[CFree[Attempt, ?]] = new Catchable[CFree[Attempt, ?]] {
+  implicit val catsFreeAttemptCatchable: Catchable[CFree[Attempt, ?]] = new Catchable[CFree[Attempt, ?]] {
```

Comment:
@peterneyens should we prefix all of our implicits with `freestyle` similar to how cats does it?